packages/web3-core/src/formatters.ts

```diff
@@ -450,7 +450,7 @@ export const outputBlockFormatter = (block: BlockInput): BlockOutput => {
 	}
 
 	if (block.baseFeePerGas) {
-		modifiedBlock.baseFeePerGas = hexToNumber(block.baseFeePerGas);
+		modifiedBlock.baseFeePerGas = outputBigIntegerFormatter(block.baseFeePerGas);
```

Comment:
This function `outputBlockFormatter` is marked for deprecated.  We can merge this but will encourage not to use it, and use format function in web3 utils. example for block formatting: https://github.com/web3/web3.js/blob/b8fa7124d1d1e4c9c16b60ad2ce58559f72efd27/packages/web3-eth/src/rpc_method_wrappers.ts#L248

Reply:
I pointed changed line but was mentioning function containing it.